### DISCUSSION TITLE
Dịch `signature`

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#168] [Terminology] [answered]
Mọi người cho ý kiến về cách dịch `signature` trong context dưới đây nhé:

![image](https://user-images.githubusercontent.com/41325216/194762263-78e2d013-6445-429a-89ba-3370d625e155.png)

cc: @mlbvn/handson-ml 
internal link: https://github.com/mlbvn/handson-ml2-book/pull/998

**Comment by cuongvng** [comment]
em nghĩ `chữ ký đầu vào` ổn

**Reply by ngcthuong** [reply]
Từ này mang ý nghĩa bản sắc riêng hơn là chữ ký

**Reply by thanhcsf** [reply]
@ngcthuong anh có gợi ý gì với từ này không ạ?

**Comment by hnguyentt** [comment]
* dấu ấn đầu vào
* đặc điểm đầu vào

**Reply by thanhcsf** [reply]
mình thấy dấu ấn, dấu hiệu có vẻ phù hợp

**Reply by rootonchair** [reply]
Em cũng thấy cách dịch này ổn

**Comment by thanhcsf** [comment, marked as answer]
Mình tạm chốt ở điểm này là **dấu ấn đầu vào** để move on nhé.
